src/_layouts/indepth_guide.erb

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+---
+
+<% @entries = site.data.guides_sidebar[resource.data.sidebar_name] %>
+<%= render Shared::Sidebar.new(data: @entries, current: resource) do |sidebar| %>
+  <% sidebar.slot :before do %>
+    <h1>
+      <%= resource.data.sidebar_title %>
+      <span class="version"><%= resource.data.sidebar_version %></span>
+    </h1>
+  <% end %>
+  <% sidebar.slot :after do %>
+    <%= render Shared::BumpCta.new(slug: site.data.slug) %>
+  <% end %>
+<% end %>
+<main class="guide-content wrapper" data-wrapper-size="small">
+  <button data-button-style="secondary" data-button-size="small" data-toggle-target="button" data-action="toggle#click">
+    Show menu <svg width="11" height="12" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="m10 6 .543.517a.75.75 0 0 0 0-1.034L10 6ZM0 6.75h10v-1.5H0v1.5Zm10.543-1.267-4.762-5-1.086 1.034 4.762 5 1.086-1.034Zm-1.086 0-4.762 5 1.086 1.034 4.762-5-1.086-1.034Z" fill="#025FD7"/></svg>
+  </button>
+  <h1><%= data.title %></h1>
```

Comment:
You are gona have 2 <h1> in the page?

Reply:
I'm not sure I understand what you mean, we always skip adding `#` in markdown content and we put the main title in the frnotmatter in case we need to retrieve it as data.

Reply:
As discussed, if we have some tags that reset the h* hierarchy, like section define in the Sidebar component, that's not a subject